### DISCUSSION TITLE
When is a model considered to be "fine-tuned?"

### DISCUSSION THREAD

**Discussion by T145** [coqui-ai/TTS#3808] [General Q&A]
![Untitled](https://github.com/coqui-ai/TTS/assets/1214129/a69a12cf-bbbb-484f-b57d-ba19351c2e49)

I've been experimenting with XTTS V2 fine-tuning, and I'm simply curious as to when the fine-tuning becomes sufficient. Most people seem to only be training their fine-tunes for 10 or so epochs while others train for thousands. Every top article online says why fine-tuning is important, what its benefits are, blah blah blah. But none seem to address issues on what you need to fix once you start getting results.

From what I understand, I want to reach a state of [convergence](https://machine-learning.paperspace.com/wiki/convergence) from the fine-tuning. What I have now shows that my learning rate is probably too high, as per this graphic:
![learning-rate](https://miro.medium.com/v2/resize:fit:640/format:webp/0*WZlfZ9Qbw18_Ysc3.jpeg)

I'd also think that my average loss should be decreasing and not increasing. Is there anywhere I'm wrong? For reference, I'm largely using the significant values from the [Coqui XTTS V2 training script](https://github.com/coqui-ai/TTS/blob/dev/tests/xtts_tests/test_xtts_v2-0_gpt_train.py), such as the learning rate. Also, I have the initial `sample_rate` set to 48000 to match my dataset.
I also saw that Coqui "recommend(s) that BATCH_SIZE * GRAD_ACUMM_STEPS needs to be at least 252 for more efficient training," so what would be their recommended values?
